src/alarmsbackendmodel_p.cpp

```diff
@@ -170,7 +170,11 @@ void AlarmsBackendModelPriv::alarmUpdated(AlarmObject *alarm)
     if (currentRow >= 0)
         alarms.removeAt(currentRow);
 
+#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
+    QList<AlarmObject*>::iterator it = std::lower_bound(alarms.begin(), alarms.end(), alarm, alarmSort);
```

Comment:
This would work for quite old Qt versions. I'd suggest just to replace qLowerBound with std::lower_bound.

Reply:
removed